test/e2e-test/rollout_trait_test.go

```diff
@@ -53,26 +52,6 @@ var _ = Describe("rollout related e2e-test,rollout trait test", func() {
 	var targerDeploy, sourceDeploy v1.Deployment
 	var err error
 
-	createAllDef := func() {
-		By("install all related definition")
-		var cd v1beta1.ComponentDefinition
-		Expect(yaml.Unmarshal([]byte(rolloutTestWd), &cd))
-		// create the componentDefinition if not exist
-		cd.Namespace = namespaceName
-		Eventually(
-			func() error {
-				return k8sClient.Create(ctx, &cd)
-			},
-			time.Second*3, time.Millisecond*300).Should(SatisfyAny(BeNil(), &util.AlreadyExistMatcher{}))
-		var td v1beta1.TraitDefinition
-		Expect(yaml.Unmarshal([]byte(rolloutTestTd), &td)).Should(BeNil())
-		td.Namespace = namespaceName
-		Eventually(func() error {
-			return k8sClient.Create(ctx, &td)
-		},
-			time.Second*3, time.Millisecond*300).Should(SatisfyAny(BeNil(), &util.AlreadyExistMatcher{}))
-	}
```

Comment:
why deleted？

Reply:
Before we write some specify component/trait definitions for these e2e-test. This due to some changes of  x-def won't be used by our e2e-tests.  Delete these means using all official x-def run our test. Guarantee  this is truly e2e-test.

Reply:
good.